### DISCUSSION TITLE
Exporting screen space stellar objects

### DISCUSSION THREAD

**Discussion by snibbe** [Stellarium/stellarium#3119] [Q&A]
Hi,
I'm interested in using Stellarium to drive some custom digital artwork and am wondering if it's possible to send screen space coordinates of planets and stars to another process via some sort of messaging (e.g. OSC)?
Thanks,
Scott

**Comment by ilkant** [comment]
Look these advices:

https://helpwiki.evergreen.edu/wiki/index.php/Astronomy_and_Cosmology_-_Create_Screenshots_and_Screencasts_and_add_to_Slide-Based_Presentation

**Reply by snibbe** [reply]
Hi, thank you @ilkant! That looks like it's a way to capture screenshots from Stellarium and turn them into a movie. What I'd like to do is to transmit the 2d screen space position of a stellar object, for example, the sun is at (312,622) in screen space. Is there any way to get that information out of Stellarium by scripting or other means?

**Comment by gzotti** [comment]
We have a RemoteControl API that communicates via HTML. Docs: See User Guide, section 13.5. 
Screen space coordinates are however not directly available.
